db/migrate/20201026181135_add_quantity_to_order_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddQuantityToOrderItems < ActiveRecord::Migration[5.2]
```

Comment:
I needed these migrations to make the order show page functional on the front end